mongo/integration/unified_spec_test.go

```diff
@@ -61,6 +62,7 @@ var (
 		"CreateCollection from encryptedFields.":           godriver2413SkipReason,
 		"DropCollection from encryptedFields":              godriver2413SkipReason,
 		"DropCollection from remote encryptedFields":       godriver2413SkipReason,
+		"Topology LifeCycle": godriver2943SkipReason,
```

Comment:
Do we have a description of "Topology LifeCycle" used in any spec test?
Also, did you run the "go fmt" to help align the key-value pair? EG "static-analysis" is complaining about the format. You may find some IDE plugins to format the lines automatically.

Reply:
Ah, it should be `Topology lifecycle`:

```
mongo/integration/unified_spec_test.go:             "Topology lifecycle":                               godriver2943SkipReason,
testdata/server-discovery-and-monitoring/unified/logging-loadbalanced.json:      "description": "Topology lifecycle",
testdata/server-discovery-and-monitoring/unified/logging-loadbalanced.yml:  - description: "Topology lifecycle"
testdata/server-discovery-and-monitoring/unified/logging-replicaset.json:      "description": "Topology lifecycle",
testdata/server-discovery-and-monitoring/unified/logging-replicaset.yml:  - description: "Topology lifecycle"
testdata/server-discovery-and-monitoring/unified/logging-sharded.json:      "description": "Topology lifecycle",
testdata/server-discovery-and-monitoring/unified/logging-sharded.yml:  - description: "Topology lifecycle"
testdata/server-discovery-and-monitoring/unified/logging-standalone.json:      "description": "Topology lifecycle",
testdata/server-discovery-and-monitoring/unified/logging-standalone.yml:  - description: "Topology lifecycle"
```